lib/test.js

```diff
@@ -111,7 +112,11 @@ Test.prototype.run = function () {
                 self.end();
             }
         })['catch'](function onError(err) {
-            self.fail(err);
+            if (objectToString(err) === '[object Error]') {
```

Comment:
I think this will only work for Error, not for the 6 builtin subclasses nor for user subclasses.

Reply:
do you have a preferred `is-error` library ?

I can also do `err instanceof Error || objectToString(err) === '[object Error]'`

`instanceof` should work for everything except cross realm which rarely happens ...

Reply:
Given that there's no robust way to identify Errors, cross-realm or otherwise, this seems like it'd be good enough.